cms/templates/howitworks.html

```diff
@@ -14,8 +16,8 @@
 <div class="wrapper-content-header wrapper">
   <section class="content content-header">
     <header>
-      <h1><span class="wrapper-text-welcome">${_("Welcome to {studio_name}").format(
-        studio_name=u'</span><span class="logo">{studio_name}</span>'.format(
+      <h1><span class="wrapper-text-welcome">${Text(_("Welcome to {studio_name}")).format(
+        studio_name=HTML(u'</span><span class="logo">{studio_name}</span>').format(
```

Comment:
I have no idea why a span with a class of logo is being used here.  I think its a mistake.  All its currently doing is adding a right margin of 10px.  When you remove the class, it looks just fine.  Removing the span and class all together would make this code so much easier to write.